### PR TITLE
add kubernetes go getting started

### PR DESCRIPTION
**Pull request by EvanBoyle**
Tested locally, works around https://github.com/pulumi/pulumi-azure/issues/530

fixes https://github.com/pulumi/docs/issues/2967
Last PR that fixes https://github.com/pulumi/home/issues/767

### CODE REVIEW

File: content/docs/get-started/kubernetes/modify-program.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ const deployment = new k8s.apps.v1.Deployment(appName, {
 const frontend = new k8s.core.v1.Service(appName, {
     metadata: { labels: deployment.spec.template.metadata.labels },
     spec: {
-        type: isMinikube ? "ClusterIP" : "LoadBalancer",
+        type: isMinikube ? "" : "LoadBalancer",
```
Comment (mikhailshilkov) [inline content/docs/get-started/kubernetes/modify-program.md:93]: Why this change?

File: content/docs/get-started/kubernetes/review-project.md [hunk 2]
```diff
@@ -93,6 +92,55 @@ deployment = Deployment(
...
+        if err != nil {
+            return err
+        }
+
+        ctx.Export("name", deployment.Metadata.Elem().Name())
+
+        return nil
+    })
+}
+```
```
Comment (mikhailshilkov) [inline content/docs/get-started/kubernetes/review-project.md:143]: Add an empty line

File: content/docs/get-started/kubernetes/review-project.md [hunk 3]
```diff
@@ -69,7 +69,6 @@ export const name = deployment.metadata.name;
 
 {{% /choosable %}}
 {{% choosable language python %}}
-
```
Comment (mikhailshilkov) [inline content/docs/get-started/kubernetes/review-project.md:72]: Restore an empty line

File: content/docs/get-started/kubernetes/modify-program.md [hunk 4]
```diff
@@ -158,6 +158,96 @@ else:
...
+                }
+                return *val.LoadBalancer.Ingress[0].Hostname
+            })
+        }
+
+        ctx.Export("ip", ip)
+        return nil
+    })
+}
+```
```
Comment (mikhailshilkov) [inline content/docs/get-started/kubernetes/modify-program.md:250]: Add an empty line

File: content/docs/get-started/kubernetes/modify-program.md [hunk 5]
```diff
@@ -158,6 +158,96 @@ else:
 pulumi.export("ip", result)
 ```
 
+{{% /choosable %}}
+{{% choosable language go %}}
+```go
```
Comment (mikhailshilkov) [inline content/docs/get-started/kubernetes/modify-program.md:163]: Add an empty line

**Comment by mikhailshilkov** [comment]
> Tested locally, works around pulumi/pulumi-azure#530

How is this related?

File: content/docs/get-started/kubernetes/modify-program.md (hunk 1 above)
Comment (EvanBoyle) [reply content/docs/get-started/kubernetes/modify-program.md:93]: A stray `ctrl+x`. Good catch

**Comment by EvanBoyle** [comment]
@mikhailshilkov the second step hits the bug that is linked (it's actually a bug in the kubernetes SDK, not the azure SDK). I have to write a few manual `.Apply` calls instead of using the kubernetes SDK builtins to work around. We should remove the apply calls from the example when the bug is fixed/released.
